core/storage/classifier/gae_models.py

```diff
@@ -95,19 +100,19 @@ def create(
                 data used by the classifier.
 
         Returns:
-            ID of the new ClassifierModel entry.
+            ID of the new ClassifierDataModel entry.
 
         Raises:
             Exception: A model with the same ID already exists.
         """
 
         instance_id = cls._generate_id(exp_id)
```

Comment:
Yep. This is what I sort of comprehended from the last, but now its clear. I've implemented those changes here.